File: level 0/자릿수 더하기.py
def solution(n):
    num = []
    for i in str(n):
        num.append(int(i))
    return sum(num)
    
print(solution(1234))
print(solution(930211))

# 더 좋은 코드
# def solution(n):
#     answer = sum(list(map(int,list(str(n))))
#     return answer

File: javascript_programmers_school/level 1/자릿수 더하기.js
function solution(n) {
  n = String(n)
    .split("")
    .map((num) => Number(num));
  return n.reduce((acc, num) => acc + num); // 초깃값으로 0을 적어주지 않으면 string이 초깃값이 된다.
}

// 더 간단히 작성하기
function solution(n) {
  // 쉬운방법
  return (n + "").split("").reduce((acc, curr) => acc + parseInt(curr), 0);
}
